pom.xml

```diff
@@ -95,8 +95,10 @@
   <properties>
     <!-- language versions -->
     <java.version>1.8</java.version>
-    <scala.version>2.10.5</scala.version>
+    <scala.2.10.version>2.10.5</scala.2.10.version>
```

Comment:
What is the reason of introducing `scala.2.10.version` and `scala.2.10.version`  instead of using just `scala.version` ?

Reply:
@zjffdu Scio & Beam interpreters need to stay on the 2.10 because for 2.11 we need to bump Beam dependencies, etc. So when compiling with `scala-2.11` profile, we need somehow specify Scala 2.10 version, and it was easier to take it from the top-level project.

Reply:
Do you mean even I specify `-Pscala-2.11` scio module still use scala-2.10 ?

Reply:
Yes, because the scio/beam dependencies are mess :-(
While Scio has both 2.10 & 2.11 dependencies, the Beam has either 2.10 or 2.11 dependencies, so it would require to switch Beam dependencies to higher versions when compiling with 2.11.

So I decided to fix the issue this way - just to fix compilation problems, etc., and then try to think how to make it more generic when working on JIRA about removing Scala 2.10 & adding Scala 2.12, etc. But that could be still a problem with Beam as it looks like that they aren't doing the releases that depend on different Scala versions.

Reply:
I see, glad to know that you plan to fix it in a followup PR. This PR LGTM